tensorflow/compiler/xla/tests/fusion_test.cc

```diff
@@ -522,6 +522,39 @@ XLA_TEST_F(FusionTest, DISABLED_ON_CPU(ReduceWindow)) {
       *ExecuteAndTransfer(std::move(hlo_module), {}));
 }
 
+XLA_TEST_F(FusionTest, SharedConstant) {
+  auto hlo_module = CreateNewModule();
```

Comment:
I'm also going to merge the master head, to eliminate the failure.  it's clearly nothing to do with adding an XLA unit test - but it would be good to have a clean test bill.